.github/workflows/release.yaml

```diff
@@ -1,28 +1,26 @@
-name: Release charm to other tracks and channels
+name: Promote charm
 
 on:
   workflow_dispatch:
     inputs:
-      destination-channel:
-        description: 'Destination Channel'
-        required: true
       origin-channel:
+        type: choice
         description: 'Origin Channel'
-        required: true
+        options:
+        - latest/edge
+      destination-channel:
+        type: choice
+        description: 'Destination Channel'
+        options:
+        - latest/stable
     secrets:
       CHARMHUB_TOKEN:
         required: true
 
 jobs:
   promote-charm:
-    name: Promote charm
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - name: Release charm to channel
-        uses: canonical/charming-actions/release-charm@1.0.3
-        with:
-          credentials: ${{ secrets.CHARMHUB_TOKEN }}
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-          destination-channel: ${{ github.event.inputs.destination-channel }}
-          origin-channel: ${{ github.event.inputs.origin-channel }}
+    uses: canonical/operator-workflows/.github/workflows/promete_charm.yaml@main
```

Comment:
Is there a typo here ?
Should be `promote_charm.yaml@main`

Reply:
Ty! correcting it right away